src/plugins/messenger/MessengerPreview/components/MessengerButtonTop.tsx

```diff
@@ -0,0 +1,14 @@
+import { MessagePluginFactoryProps } from '../../../../common/interfaces/message-plugin';
+
+export const getMessengerButtonTop = ({ React, styled }: MessagePluginFactoryProps) => {
+    const MessengerFrame = styled.div(({ theme }) => ({
+        width: 250,
+        borderRadius: theme.unitSize * 2,
+        overflow: 'hidden',
+        background: theme.primaryGradient,
+        color: theme.primaryContrastColor,
```

Comment:
Looks nice with the theming, but why not add it everywhere then (lists, galleries)? Also, we should probably make this an optional config setting as it changes previous behavior

Reply:
Lists and Galleries are different Buttons are more like QRs. I consider it a bug before. Let's keep only here.